client/src/component/footerBuyer/style.css

```diff
@@ -0,0 +1,60 @@
+.footer {
+  position: fixed;
```

Comment:
it's better not to use fixed position, it will make the footer always visible and take a lot of screens.

Reply:
- footer will be fixed at the end of the page!!